monkeys/src/main/kotlin/com/wire/kalium/monkeys/actions/SendMessageAction.kt

```diff
@@ -18,11 +18,60 @@
 package com.wire.kalium.monkeys.actions
 
 import com.wire.kalium.logic.CoreLogic
+import com.wire.kalium.monkeys.conversation.MonkeyConversation
 import com.wire.kalium.monkeys.importer.ActionType
 import com.wire.kalium.monkeys.importer.UserCount
+import com.wire.kalium.monkeys.logger
+import com.wire.kalium.monkeys.pool.ConversationPool
+import com.wire.kalium.monkeys.pool.MonkeyPool
+
+private const val ONE_2_1: String = "One21"
```

Comment:
This could be an `enum` for different targets, instead of a hardcoded string. It's also simple to (de)serialise using KotlinX Serialization when importing the configuration.